bokeh/models/glyph.py

```diff
@@ -120,6 +120,26 @@ class ConnectedXYGlyph(XYGlyph):
 
     '''
 
+@abstract
+class LineGlyph(Glyph):
+    ''' Glyphs with line properties
+    '''
```

Comment:
When introducing such classes, it would be reasonable to move related properties from concrete glyph models to here, so that we would have:
```py
@abstract
class LineGlyph(Glyph):
    ''' Glyphs with line properties
    '''

    fill_props = Include(ScalarFillProps, use_prefix=False, help="""
    The %s values for the horizontal directed area.
    """)
```
However, then we would have to provide a mechanism to fill in glyph names (like `area` in this case, as I copied this from `HArea` glyph), so it may be more trouble than it's worth at this point. Though if we get to statically typing this code with mypy, then we may be further encouraged to do so by the type-checker.

Reply:
I considered this, and it does seem natural, but I was nervous about messing around too much with the properties system that I don't fully understand.  As it is,  it is inelegant that I had to match the mixin classes with the properties "by hand."